### PR TITLE
Add support for offline indexing

### PR DESCRIPTION
**Pull request by WadeBarnes**

### CODE REVIEW

File: openshift/manage [hunk 1]
```diff
@@ -135,6 +135,195 @@ usage () {
...
+          <searchEnginePodName/>
+            - Optional - The name of the pod hosting the offline search engine service.
+          <dbPodName/>
+            - Optional - The name of the pod hosting the offline database.
+        Examples;
+          $0 -p bc-offline-indexing -e test scaleOfflineIndexEnv up
+          $0 -p bc-offline-indexing -e test scaleOfflineIndexEnv down
+
+    swapIndexStorage [<onlineSearchEngine/>] [<offlineSearchEngine/>]
+      - Swap the index storage between the online nad offline search engines.
```
Comment (esune) [inline openshift/manage:289]: Typo: `nad` -> `and`

File: openshift/manage [hunk 2]
```diff
@@ -611,6 +869,320 @@ function locateBadRecord() {
...
+    fi
+
+    # Fetch the volume info
+    onlineSearchEngineVolume=$(getVolume "${_onlineSearchEngine}" "${_projectName}")
+    onlineSearchEnginePvc=$(getClaimName "${onlineSearchEngineVolume}")
+    offlineSearchEngineVolume=$(getVolume "${_offlineSearchEngine}" "${_projectName}")
+    offlineSearchEnginePvc=$(getClaimName "${offlineSearchEngineVolume}")
+
+    echoWarning \\n"Container - ${_onlineSearchEngine}:"
+    echoWarning "  - PCV: ${onlineSearchEnginePvc}"
```
Comment (esune) [inline openshift/manage:938]: Typo `PCV` -> `PVC`

**Review by esune — CHANGES_REQUESTED** [review]
Just a couple of typos to tweak, looks good otherwise.

File: openshift/manage (hunk 1 above)
Comment (WadeBarnes) [reply openshift/manage]: fixed

File: openshift/manage (hunk 2 above)
Comment (WadeBarnes) [reply openshift/manage]: fixed

**Review by esune — APPROVED** [review]

**Pull request merged**